test/conftest.py

```diff
@@ -0,0 +1,112 @@
+import json
+import os
+import pytest
+import requests
+import subprocess
+import time
+
+
+# settings are coming from the docker-compose file
+
+_COMPOSE_FILE = "docker-compose.yaml"
+_COMPOSE_PROJECT_NAME = "auth_client_tests"
+_AUTH_SERVICE_NAME = "auth"
+
+AUTH_URL = "http://localhost:50001/testmode"
+_AUTH_API = AUTH_URL + "/api/V2/"
+AUTH_VERSION = "0.7.1"
+
+SOME_RANDOM_ROLE1 = "random1"
+SOME_RANDOM_ROLE2 = "random2"
+
+
+def _wait_for_services(timeout: int = 30):
+    start = time.time()
+    while time.time() - start < timeout:
+        try:
+            res = subprocess.run(
+                [
+                    "docker", "compose",
+                    "-f", _COMPOSE_FILE, "-p", _COMPOSE_PROJECT_NAME,
+                    "ps", "--format", "json"],
+                capture_output=True,
+                check=True,
+            )
+            status = None
+            for jsonline in res.stdout.decode("utf-8").split("\n"):
+                if jsonline.strip():
+                    j = json.loads(jsonline.strip())
+                    if j["Service"] == _AUTH_SERVICE_NAME:
+                        status = j["Health"]
+                        if status == "healthy":
```

Comment:
could just check the content of `j["Health"]` without saving it to a variable -- `if j.get("Health") ...`